lnbits/nodes/lndrest.py

```diff
@@ -167,6 +167,8 @@ async def close_channel(
         point: Optional[ChannelPoint] = None,
         force: bool = False,
     ):
+        if short_id:
```

Comment:
did `ruff` suggested this? how can it know?

Reply:
ups, that was a pyright, it was an unused local variable